ietf-layer0-types-ext.yang

```diff
@@ -923,4 +966,28 @@ module ietf-layer0-types-ext {
       description "Wavelength Allocation Method";
     }
   }
+
+  grouping frequency-range {
+    description
+      "The parameters that define a frequency range.";
+    leaf lower-frequency {
+      type frequency-thz;
+      mandatory true;
+      description
+        "The lower frequency boundary of the
+        frequency range.";
+    }
+    leaf upper-frequency {
+      type frequency-thz;
+      must '. > ../lower-frequency' {

```

Comment:
is it supposed to be : must "current() > ../lower-frequency"

Reply:
Why? Isn't current() typically used in a list to returns a node set with the initial context node ?

Reply:
When defining a grouping you will not know whether this grouping is used in a list or not. Therefore it is always safer to use "current()" as the current() function just returns the current node.

Reply:
Italo/Aihua: we can keep the current must statement. Aihua will further investigate and open a new issue on github if needed.